[TT-4744] (#1892)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-stack/tyk-multi-data-centre/setup-master-data-centre.md

```
## Installing in a Kubernetes Cluster with our Helm Chart

> If you are deploying the **Tyk Control plane**, a.k.a **MDCB**, for a **Tyk Multi Data Centre Bridge** deployment then you set the `mdcb.enabled: true` option in the local `values.yaml` to add of the **MDCB** component to your installation.

Reviewer: ```suggestion
Deploying the MDCB ( Master Data Centre Bridge) component enables multi-cluster, multi-data-centre API management from a single Dashboard.

To deploy the **MDCB** ( Master Data Centre Bridge) component in your Kubernetes cluster, set `mdcb.enabled` in your `values.yaml` to `true`.

For more information related to Tyk Helm charts, check [Tyk Helm Chart]({{< ref "/content/tyk-on-prem/installation/kubernetes/helm-chart.md" >}})
```

> Check [Tyk Control plane](https://tyk.io/docs/tyk-multi-data-centre/) for more configuration details.

Reviewer: ```suggestion
Check [Tyk Control plane]({{< ref "/content/tyk-multi-data-centre/" >}}) for more configuration details.
```

> This setting enables multi-cluster, multi Data-Centre API management from a single dashboard. For more information, you can go to [Tyk Helm Chart]({{< ref "/content/tyk-on-prem/installation/kubernetes/helm-chart.md" >}})

Reviewer: ```suggestion
```
